test/unit/ngsiv2/general/deviceService-test.js

```diff
@@ -223,7 +314,7 @@ describe('NGSI-v2 - Device Service: utils', function () {
                 .post('/v2/entities?options=upsert')
                 .reply(204);
 
-            async.series([utils.request.bind(utils.request, groupCreation)], function (error, results) {
+            async.series([request.bind(request, configGroupCreation)], function (error, results) {
```

Comment:
Those changes were not included in PR #954. Could you elaborate why are included here?